src/HttpClient.php

```diff
@@ -21,7 +21,8 @@
      *
      * @return ResponseInterface
      *
-     * @throws Exception
+     * @throws Exception  If an error happens during processing the request.
+     * @throws \Exception If processing the request is impossible (eg. bad configuration).
```

Comment:
agreed for this situation. but in general we should keep using the `use` statement for phpdoc as well.